astroquery/heasarc/tests/setup_package.py

```diff
@@ -1,11 +0,0 @@
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
-
-
-import os
-
-
-def get_package_data():
-    paths = [
-        os.path.join("data", "*.dat"),
```

Comment:
I don't see the datafiles being deleted. Should they be?

Reply:
Removed the data/ directory now. Thanks for the catch.